bundles/redhat8.8/packages.txt.gotmpl

```diff
@@ -36,6 +36,6 @@ gssproxy
 libverto-module-base
 libverto
 {{ if .FetchKernelHeaders -}}
-kernel-headers-4.18.0-477.58.1.el8_8
-kernel-devel-4.18.0-477.58.1.el8_8
+kernel-headers-4.18.0-477.89.1.el8_8
```

Comment:
this isn't supported anymore

Reply:
I'm not sure I understand, what is not supported?
(keep in mind this is on a release branch)